src/test/java/org/prebid/server/auction/BasicCategoryMappingServiceTest.java

```diff
@@ -307,9 +312,10 @@ public void applyCategoryMappingShouldRejectBidsWithCatLengthMoreThanOne() {
         assertThat(resultFuture.result().getBiddersToBidsCategories())
                 .isEqualTo(Collections.singletonMap(givenBid("1", null, "10", singletonList("cat1")),
                         "10.00_fetchedCat1_10s"));
-        assertThat(resultFuture.result().getErrors()).hasSize(1)
-                .containsOnly("Bid rejected [bidder: otherBid, bid ID: 2] with a reason: Bid has more than one"
-                        + " category");
+        assertThat(resultFuture.result().getPrebidLog().error().getAllMessages())
+                .containsOnly(PrebidMessage.of(10009,
```

Comment:
Please, replace with call to `containsExactly`. Also, please, check for similar occurences.